src/lib/controllers/DeviceController.js

```diff
@@ -3,12 +3,13 @@
 import type { Device, DeviceRepository } from '../../types';
 import type { DeviceAPIType } from '../deviceToAPI';
 
+
 import Controller from './Controller';
 import httpVerb from '../decorators/httpVerb';
 import route from '../decorators/route';
 import deviceToAPI from '../deviceToAPI';
 
-class DevicesController extends Controller {
+class DeviceController extends Controller {
```

Comment:
This should be `DevicesController`.  I want all the controllers to be plural since the API is `/v1/devices`